app/views/customers/_deleteCustomerModal.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="modal fade" id="deleteCustomerModal<%= customer.id %>" tabindex="-1" aria-labelledby="deleteCustomerModalLabel<%= customer.id %>" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="deleteCustomerModalLabel<%= customer.id %>">Delete Customer</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <p>Are you sure you want to delete the customer: <strong><%= customer.full_name %></strong>?</p>
+      </div>
+      <div class="modal-footer">
+        <%= link_to 'Delete', customer_path(customer), method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger' %>
```

Comment:
The modal section of the lesson comes with a custom endpoint that deletes the customers' orders as well, but because that section is optional anyway, skipping the custom endpoint is okay.